File: examples/university/src/main/java/entity/Address.java
package entity;

import jakarta.persistence.Embeddable;
import lombok.Getter;
import lombok.Setter;

@Embeddable
@Getter
@Setter
public class Address {
	
	private String street;
	
	private String city;
	
	private Integer postCode;
	
	private String country;
	
}

File: orm-reference-implementations/entity-framework/entity/Address.cs
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations.Schema;

namespace csharp_example.entity
{
    [Owned]
    public class Address
    {
        [Column(TypeName = "Varchar(255)")]
        public string? Street { get; set; }


        [Column(TypeName = "Varchar(255)")]
        public string? City { get; set; }

        public int? PostCode { get; set; }


        [Column(TypeName = "Varchar(255)")]
        public string? Country { get; set; }
    }
}